pytest_fixtures/api_fixtures.py

```diff
@@ -303,6 +305,34 @@ def module_azurerm_cloudimg(default_architecture, default_os, module_azurerm_cr)
     return finish_image
 
 
+@pytest.fixture(scope='module')
+def module_azurerm_gallery_finishimg(default_architecture, default_os, module_azurerm_cr):
```

Comment:
`module_azurerm_gallery_finishimg >> module_azurerm_gallery_finish_img` what do you think ?

Reply:
Using the same naming to keep consistent as others already have ..

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L264

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L27